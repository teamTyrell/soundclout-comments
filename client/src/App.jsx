import React from 'react';
import './App.css';

class App extends React.Component {
  constructor(props) {
    super(props);

    this.state = {

    };
  }

  render() {
    return (
      <>
        Hello World
      </>
    );
  }
}

export default App;