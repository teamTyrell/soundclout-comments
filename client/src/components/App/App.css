
.App__container {
  /* border: 1px solid black; */
}

.App__comments-heading {
  height: 89px;
  border-bottom: 1px solid #F2F2F2;
  margin-bottom: 20px;
  /* border: 1px solid black; */
}

.App__comment-input-container {
  margin-bottom: 12px;
}

.App__song-actions-container {
  display: flex;
  justify-content: space-between;
  align-items: center;
  width: 100%;
}

.App__song-actions-buttons {
  display: flex;
  align-items: center;
}

.App__song-actions-buttons > button {
  margin-right: 5px;
}

.App__button-content {
  display: flex;
  align-items: center;
}

.App__button-content > span {
  margin-left: 8px;
  color: #333;
}

.App__like-icon {
  font-size: 1.3rem;
}

.App__plays-icon {
  font-size: 1.1rem;
  color: #999;
  margin-right: 6px;
}

.App__likes-icon {
  font-size: 1.1rem;
  color: #999;
  margin-right: 4px;
}

.App__reposts-icon {
  font-size: 1.3rem;
  color: #999;
  margin-right: 4px;
}

.App__song-actions-info {
  display: flex;
  /* justify-content: space-between; */
  align-items: center;
  /* min-width: 152px; */
}

.App__song-info {
  cursor: pointer;
  margin-left: 12px;
}

.App__song-info > span {
  font-family: Interstate, "Lucida Grande", Garuda, Verdana, Tahoma, sans-serif;
  font-size: 1.3rem;
  font-weight: 300;
  color: #999;
}

.App__song-info:hover > span {
  color: #333;
}

.App__comments-body {
  display: flex;
}

.App__artist-container {
  width: 148px;
  height: 400px;
}

.App__comments-container {
  flex: 1;
}

.App__comments-meta-info {
  margin: 20px 0;
}

.App__comments-meta-info--show-less {
  height: 120px;
  overflow: hidden;
}

/* .App__comments-meta-info--show-less:before {
  content: " ";
  display: block;
  margin-top: 142px;
  margin-bottom: -162px;
  height: 20px;
  width: 100%;
  background: #fff;
  background: linear-gradient(hsla(0,0%,100%,0),hsla(0,0%,100%,.5) 90%,#fff);
  z-index: 1;
} */

.App__song-description {
  font-family: Interstate, "Lucida Grande", Garuda, Verdana, Tahoma, sans-serif;
  font-size: 1.5rem;
  line-height: 20px;
  font-weight: 300;
  color: #333;
  margin-bottom: 10px;
}

.App__released-by {
  margin: 10px 0;
}

.App__released-by > h4 {
  font-family: Interstate, "Lucida Grande", Garuda, Verdana, Tahoma, sans-serif;
  font-size: 1.5rem;
  font-weight: 400;
  color: #333;
  margin-bottom: 4px;
}

.App__released-by > span {
  font-family: Interstate, "Lucida Grande", Garuda, Verdana, Tahoma, sans-serif;
  font-size: 1.5rem;
  font-weight: 300;
  color: #333;
}

.App__release-date {
  margin: 10px 0;
}

.App__release-date > h4 {
  font-family: Interstate, "Lucida Grande", Garuda, Verdana, Tahoma, sans-serif;
  font-size: 1.5rem;
  font-weight: 400;
  color: #333;
  margin-bottom: 4px;
}

.App__release-date > span {
  font-family: Interstate, "Lucida Grande", Garuda, Verdana, Tahoma, sans-serif;
  font-size: 1.5rem;
  font-weight: 300;
  color: #333;
}

.App__explicit {
  font-family: Interstate, "Lucida Grande", Garuda, Verdana, Tahoma, sans-serif;
  font-size: 1.5rem;
  font-weight: 400;
  color: #333;
  margin-bottom: 10px;
}

.App__show-more-button {
  border: none;
  outline: none;
  background-color: transparent;
  margin: 0;
  padding: 0;
  font-family: "Lucida Grande", Garuda, Verdana, Tahoma, sans-serif;
  font-size: 1.2rem;
  font-weight: 400;
  color: #999;
  cursor: pointer;
  /* margin-top: -4px; */
  margin-bottom: 20px;
  width: 100%;
  text-align: left;
}

.App__show-more-button:hover {
  color: #333;
}