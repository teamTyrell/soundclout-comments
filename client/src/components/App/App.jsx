import React from 'react';
import './App.css';
import {
  ArtistInfo,
} from '../components';

class App extends React.Component {
  constructor(props) {
    super(props);

    this.state = {

    };
  }

  render() {
    return (
      <div className='App__container'>

        <div className='App__comments-heading'>

        </div>

        <div className='App__comments-body'>

          <div className='App__artist-container'>
            <ArtistInfo />
          </div>

          <div className='App__comments-container'>

          </div>

        </div>

      </div>
    );
  }
}

export default App;