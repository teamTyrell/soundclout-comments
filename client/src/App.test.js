import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

describe('<App />', () => {

  it('renders <App />', () => {

    const component = renderer.create(<App />);

    expect(component.toJSON()).toMatchSnapshot();

  });

});
