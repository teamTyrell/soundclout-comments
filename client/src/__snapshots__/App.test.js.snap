// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders <App /> 1`] = `
<div
  className="App__container"
>
  <div
    className="App__comments-heading"
  />
  <div
    className="App__comments-body"
  >
    <div
      className="App__artist-container"
    />
    <div
      className="App__comments-container"
    />
  </div>
</div>
`;
